scripts/patrol-gui.lua

```diff
@@ -91,6 +100,9 @@ local function dropdown_index_lookup(index, spidertron)
   if is_maraxsis_submarine(spidertron) then
     table.insert(lookup, "submerge")
   end
+  if is_lex_aircraft(spidertron) then
+    table.insert(contents, { "gui-patrol.liftoff" })
+  end
```

Comment:
That does not look at all right - this should be in dropdown_contents() and something else should be here.

Reply:
Copied the wrong code, fixed.

Reply:
I'm pretty sure you still need some that code in `dropdown_contents()` though...